### ISSUE TITLE
Best trained model giving suboptimal accuracy.

### ISSUE THREAD

**Issue by AmanSavaria1402** [iMoonLab/MeshNet#19]
Thank you for designing and publishing such a well-documented repo for the paper. I am not able to reproduce the model performance from the paper from the pre-trained model weights as well and training one from scratch.
For the pre-trained model, I am able to get an accuracy of around 45% and when I train the model, the accuracy gets saturated at around 45% without much improvement.

I know that the training procedure used by the authors is different and is not given in the repo but my results are very low.
Any help would be much appreciated.

Thank You.
